### PR TITLE
[benchmark]add emotion_analysis code in benchmark

### PR DESCRIPTION
**Pull request by shenx3x**
add emotion_analysis code in benchmark

### CODE REVIEW

File: benchmark/utils_em.js [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    let width = imageElement.naturalWidth;
+    let height = imageElement.naturalHeight;
+    canvas.setAttribute('width', width);
+    canvas.setAttribute('height', height);
+    let ctx =canvas.getContext('2d');
+    ctx.drawImage(imageElement,0,0,width,height);
+    this.imageElement = document.createElement('img');
+    this.imageElement.setAttribute('src',canvas.toDataURL());
+  }
+
```
Comment (BruceDai) [inline benchmark/utils_em.js]: @shenx3x  Please remove redundant blank lines.

File: benchmark/utils_em.js [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+    const mean = preOptions.mean || [0, 0, 0, 0];
+    const std = preOptions.std  || [1, 1, 1, 1];
+    const norm = preOptions.norm || false;
+    // const channelScheme = preOptions.channelScheme || 'RGB';
+    this.inputTensor = new Float32Array(this.modelInfoDict.inputSize.reduce((a, b) => a * b));
+    this.outputTensor = new Float32Array(this.modelInfoDict.outputSize);
+    inputCanvas.setAttribute("width", width);
+    inputCanvas.setAttribute("height", height);
+    let canvasContext = inputCanvas.getContext('2d');
+    canvasContext.drawImage(imageElement, box[0], box[2],
```
Comment (BruceDai) [inline benchmark/utils_em.js]: @shenx3x Please update some issues of coding style

File: benchmark/utils_ea.js [hunk 3]
```diff
@@ -0,0 +1,216 @@
+// Benchmark for Facial Landmark Detection models
```
Comment (BruceDai) [inline benchmark/utils_ea.js]: @shenx3x Please update this comment.

File: benchmark/utils_ea.js [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+    } else {
+      imageElement.src = bkImageSrc;
+    }
+    let canvas = document.createElement('canvas')
+    let width = imageElement.naturalWidth;
+    let height = imageElement.naturalHeight;
+    canvas.setAttribute('width', width);
+    canvas.setAttribute('height', height);
+    let ctx = canvas.getContext('2d');
+    ctx.drawImage(imageElement,0,0,width,height);
```
Comment (miaobin) [inline benchmark/utils_ea.js]: @shenx3x Please modify some issues of coding style.

File: benchmark/utils_ea.js [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+  async setupFaceDetector() {
+    let model = faceDetectionModels.filter(f => f.modelName == this.modelName);
+    model[0].modelFile = '../examples' + model[0].modelFile.slice(2);
+    await this.faceDetector.loadModel(model[0]);
+    await this.faceDetector.init(this.backend, preferSelect.value);
+    model[0].modelFile = '..' + model[0].modelFile.slice(11);
+  }
+
+  async getFaceDetectResult() {
+    let detectResult = await this.faceDetector.getFaceBoxes(this.imageElement);
```
Comment (miaobin) [inline benchmark/utils_ea.js]: @shenx3x this.imageElement is undefined in constructor.

**Comment by ibelem** [comment]
Merge this code.

**Pull request merged**